Sources/KeyboardShortcuts/KeyboardShortcuts.swift

```diff
@@ -179,6 +204,12 @@ public enum KeyboardShortcuts {
 
 		legacyKeyDownHandlers = [:]
 		legacyKeyUpHandlers = [:]
+
+		// invalidate and remove all elements of userDefaultsObservers
```

Comment:
This should comment that it's "defensive" since it's already done when calling `removeAll()` (deinit).